src/main/java/medicaldoctor/entities/User.java

```diff
@@ -39,7 +39,10 @@
 
     @Column(name = "UserName", nullable = false, length = 50, unique = true)
     private String userName;
-
+    
+    @Column(name = "Email", nullable = false, length = 50, unique = true)
+    private String email;
```

Comment:
please look at what  you are copying, this is the third time you are  adding a entity field and not removing the "unique".
I admit this *is* different, because emails should be unique, but we don't need an index on this field

Reply:
use `unique` only if we are commonly querying by this column, like usernames